homeassistant/bootstrap.py

```diff
@@ -118,7 +118,7 @@ def _setup_component(hass: core.HomeAssistant, domain: str, config) -> bool:
 
         # Assumption: if a component does not depend on groups
         # it communicates with devices
-        if group.DOMAIN not in getattr(component, 'DEPENDENCIES', []):
+        if 'group' not in getattr(component, 'DEPENDENCIES', []):
```

Comment:
It's not prettier, I'm good with these changes.

On Tue, Sep 6, 2016 at 8:13 PM, Paulus Schoutsen <notifications@github.com>
wrote:

> In homeassistant/bootstrap.py
> <https://github.com/home-assistant/home-assistant/pull/3203#discussion_r77750626>
> :
>
> > @@ -118,7 +118,7 @@ def _setup_component(hass: core.HomeAssistant, domain: str, config) -> bool:
> >
> >          # Assumption: if a component does not depend on groups
> >          # it communicates with devices
> > -        if group.DOMAIN not in getattr(component, 'DEPENDENCIES', []):
> > +        if 'group' not in getattr(component, 'DEPENDENCIES', []):
>
> This would be the alternative, not sure if prettier?
>
> group = loader.get_component('group')
> if group.DOMAIN not in getattr(component, 'DEPENDENCIES', []):
>     …
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3203/files/6232ed440ba416b98afad51dc1353c3b16ad7a79#r77750626>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZddcglEfGxJ8cne4QoHYJ9VZUT3aks5qnh3AgaJpZM4J1Cl0>
> .
>